src/cmd/run.go

```diff
@@ -57,6 +69,41 @@ func doRun(cmd *cobra.Command, args []string) {
 	client.RunnerUnregister(&runner.Id)
 }
 
+func electLeader() {
+	leaseLockName := "opslevel-runner-leader-lock"
+	leaseLockNamespace := viper.GetString("pod-namespace")
```

Comment:
is this the namespace we want?  This is used for where the job pods will spin up - which generally is different then where the opslevel-runner is running.